README.md

```diff
@@ -29,6 +29,24 @@ docker-compose exec web /bin/sh -c "bundle exec rails db:setup"
 
 Then open http://localhost:3000 to see the app.
 
+## Setting up the app for local (non-Docker) development
+
+### Installing prerequisites
+
+On Linux:
+
+```bash
+sudo apt install postgresql-9.6 postgresql-contrib-9.6
```

Comment:
A requirement of the project is to have Postgres installed so I don’t think we need to specify this step for Linux users, we aren’t telling OSX users how to install Postgres....

Reply:
I think the issue @timabell ran into was that he didn't realize he needed `postgresql-contrib-*`, and the error that comes up doesn't make it obvious.

The other thing we don't make clear is that we use PostgreSQL 9.6 in production, so that's what we should have installed in dev. That's not necessary when using the Docker route, of course, but all this is moot if we only worry about that.

Reply:
I’ll let this go but I’d like to fix up the reader afteer this is merged as we lost the non-docker setup and prerequisites when the docker stuff was merged it. I didn’t realise that

Reply:
Not your fault @tvararu